packages/editor/src/components/inserter-sidebar/index.js

```diff
@@ -30,23 +30,14 @@ export default function InserterSidebar( {
 		const { getInserterSidebarToggleRef, getInsertionPoint } = unlock(
 			select( editorStore )
 		);
-		const { getBlockRootClientId, __unstableGetEditorMode, getSettings } =
-			select( blockEditorStore );
+		const { getSettings } = select( blockEditorStore );
 		const { get } = select( preferencesStore );
-		const getBlockSectionRootClientId = () => {
-			if ( __unstableGetEditorMode() === 'zoom-out' ) {
-				const { sectionRootClientId } = unlock( getSettings() );
-				if ( sectionRootClientId ) {
-					return sectionRootClientId;
-				}
-			}
-			return getBlockRootClientId();
```

Comment:
This is a useless call without passing a selected block, it always returns `''`.